src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -175,18 +177,23 @@ def event_market_cycle(self):
             self._reset_event_tick_counter()
             market_event_channel = f"{self.channel_prefix}/events/market"
             current_market_info = self.market.info
+            if self.is_aggregator_controlled:
+                current_market_info.update(default_market_info)
             current_market_info['device_info'] = self._device_info_dict
             current_market_info["event"] = "market"
             current_market_info["area_uuid"] = self.device.uuid
             current_market_info['device_bill'] = self.device.stats.aggregated_stats["bills"] \
                 if "bills" in self.device.stats.aggregated_stats else None
-            current_market_info['last_market_stats'] = \
-                self.market_area.stats.get_price_stats_current_market()
+            current_market_info["last_market_maker_rate"] = \
```

Comment:
```suggestion
            current_market_info["current_market_maker_rate"] = \
```
in order to have uniform name as the method being called here